pkg/platform/model/buildplanner/impactreport.go

```diff
@@ -27,9 +32,26 @@ func (b *BuildPlanner) ImpactReport(params *ImpactReportParams) (*response.Impac
 		return nil, errs.Wrap(err, "Unable to marshal buildexpression")
 	}
 
-	request := request.ImpactReport(params.Owner, params.Project, beforeExpr, afterExpr)
+	var resp *response.ImpactReportResponse
+	for i := 0; i < numRetries; i++ {
+		resp, err = run(b.client, params.Owner, params.Project, beforeExpr, afterExpr)
+		if err == nil || !response.IsImpactReportBuildPlanningError(err) {
+			break
+		}
+		logging.Debug("Impact report response was that the buildplanner was still planning; trying again")
+		time.Sleep(retryInterval * time.Duration(i+1))
+	}
+	if err != nil {
+		return nil, errs.Wrap(err, "failed to get impact report")
```

Comment:
The error will look like this:

```
█ Something Went Wrong

 x Command failed due to unexpected error. For your convenience, this is the error chain:
  Failed to fetch impact report:
      failed to get impact report: |-
          Could not resolve one or both input buildplans
          buildAfter status: BuildPlanning.
```

How should we present this to the user? In my experience, retrying the command worked.

Reply:
Let's make this user a user facing error that reads:

> Timed out waiting for build to resolve, please try again in a few minutes.